.flake8

```diff
@@ -77,11 +77,10 @@ per-file-ignores =
     python/grass/temporal/temporal_topology_dataset_connector.py: E722
     # Current benchmarks/tests are changing sys.path before import.
     # Possibly, a different approach should be taken there anyway.
-    python/grass/pygrass/tests/benchmark.py: E402, F821
+    python/grass/pygrass/tests/benchmark.py: F821
     # Configuration file for Sphinx:
     # Ignoring import/code mix and line length.
     # Files not managed by Black
-    python/grass/imaging/images2gif.py: E226
```

Comment:
At the next PR, I think you could have a clean up of the comments of sections that are now empty.